pkgs/development/python-modules/metakernel/default.nix

```
@@ -0,0 +1,32 @@
+{ lib,
+  buildPythonPackage,
+  fetchPypi,
+  ipykernel,
+  isPy27,
+  mock,
+  pytest
+}:
+
+buildPythonPackage rec {
+  pname = "metakernel";
+  version = "0.27.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0aqq9zil6h7kxsg3v2008nr6lv47qvcsash8qzmi1xh6r4x606zy";
+  };
+
+  buildInputs = [ipykernel] ++ lib.optional isPy27 [mock pytest];
+  propagatedBuildInputs = [ipykernel];
```

```suggestion
  propagatedBuildInputs = [ ipykernel ];
```

Python 2 is no longer supported and has a different file.